packages/dds/task-manager/src/test/taskManager.spec.ts

```diff
@@ -29,8 +29,36 @@ function createConnectedTaskManager(id: string, runtimeFactory: MockContainerRun
     return taskManager;
 }
 
-const createLocalTaskManager = (id: string) =>
-    new TaskManager(id, new MockFluidDataStoreRuntime(), TaskManagerFactory.Attributes);
+function createDetachedTaskManager(id: string, runtimeFactory: MockContainerRuntimeFactory):
+    { taskManager: TaskManager; attach: () => void; } {
```

Comment:
Nit, should there be a naming update?  createDetachedTaskManager and createLocalTaskManager sound like the same thing.

Reply:
I wonder if we can just delete the local task manager test all together, since the only thing it tests (creating a local task manager) is covered by the detached tests

Reply:
@ChumpChief, I decided to remove that test and `createLocalTaskManager()` all together. Let me know if you think we should revert that and keep it in!

Reply:
Seems fine - I usually just include a little test like that so that if someone screws something up far away (e.g. in ContainerRuntime or something), "Can create a TaskManager" failing is probably a clearer signal to them than something more scenario-specific ("Can volunteer for a task" failed).  But "Can create a connected TaskManager" also fills this role sufficiently well I think.